packages/manager/modules/mfa-enrollment/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "@ovh-ux/mfa-enrollment",
+  "version": "0.0.0",
+  "private": true,
+  "license": "BSD-3-Clause",
```

Maybe you could add the `author` field:
```suggestion
  "license": "BSD-3-Clause",
  "author": "OVH SAS",
```